client/battle/BattleWindow.cpp

```diff
@@ -256,15 +256,15 @@ void BattleWindow::createTimerInfoWindows()
 		if (attacker.isValidPlayer())
 		{
 			if (ENGINE->screenDimensions().x >= 1000)
```

Comment:
I think we can now reduce this requirement a bit? Would have been great to squeeze it into 960px (which is screen width on 16x9 with maxed-out scaling)